docs/_data/portland-2022-sessions.yaml

```diff
@@ -102,7 +102,7 @@
   - name: Kate Mueller
     slug: kate-mueller
     twitter: 
-    website: Can I add this later? Site is under development...
+    website: http://www.livewithsass.com/
```

Comment:
LOL

Reply:
That's what we get for not doing external link checking ;)

Reply:
Maybe at least check for `^http`? That way we can manually fix up